hack/local-up-cluster.sh

```diff
@@ -363,7 +363,7 @@ function start_kubelet {
       if [[ -n "${NET_PLUGIN}" ]]; then
         net_plugin_args="--network-plugin=${NET_PLUGIN}"
       fi
-      
+
```

Comment:
Will do, I need a modification to this file to build and I keep accidentally committing parts of it.